modules/project/variables.tf

```diff
@@ -78,12 +78,19 @@ variable "descriptive_name" {
   default     = null
 }
 
+variable "default_alerts_email" {
+  description = "Default email address for alerting."
+  type        = string
+  default     = null
+}
+
 variable "factories_config" {
   description = "Paths to data files and folders that enable factory functionality."
   type = object({
-    custom_roles = optional(string)
-    org_policies = optional(string)
-    quotas       = optional(string)
+    custom_roles           = optional(string)
+    org_policies           = optional(string)
+    quotas                 = optional(string)
+    logging_metrics_alerts = optional(string)
```

Comment:
I'd prefer to have this split in 3 separate factories: `metrics`, `alerts`, `notification_channels`